src/pyFAI/gui/pilx/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.0.4"
```

Comment:
I think this should be removed ?

Reply:
Is it used anywhere ?
If not it could be removed.

Reply:
Not it is not used. It can be removed.